CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 1.7
+
+**Bugs**
+
+* [#48](https://github.com/resourcelib/resourcelib/pull/56) Fixed IconDirectoryResource.SaveTo has bugs with some .ico file
```

Comment:
Make this the same format as other lines please, with your name, dashes and colons, please.